src/Eshopworld.Tests.Core/EnumExtensions.cs

```diff
@@ -30,5 +32,16 @@ public static class EnumExtensions
             return oneExceptions.Select(e => typeof(T1).FullName + "." + e)
                                 .Concat(twoExceptions.Select(e => typeof(T2).FullName + "." + e));
         }
+
+        /// <summary>
+        /// Returns a random value from the enumeration <see cref="Type"/>.
+        /// </summary>
+        /// <typeparam name="T">The <see cref="Type"/> of the enumeration that we are randomizing.</typeparam>
+        /// <returns>A random value from the enumeration <see cref="Type"/></returns>
+        public static T Random<T>()
+        {
+            var values = Enum.GetValues(typeof(T));
+            return (T)values.GetValue(Rng.Next(values.Length));
```

Comment:
Not thread-safe but it shouldn't matter.

Reply:
The only risk would be corruption of the `_seedArray` in the Random object which would affect distribution. But in this case it would cause no impact.